network-store-client/src/main/java/com/powsybl/network/store/client/RestNetworkStoreClient.java

```diff
@@ -31,7 +33,7 @@ public class RestNetworkStoreClient implements NetworkStoreClient {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(RestNetworkStoreClient.class);
 
-    private static final int RESOURCES_CREATION_CHUNK_SIZE = 1000;
+    private static final int RESOURCES_CREATION_CHUNK_SIZE = 10000;
```

Comment:
Is this on purpose or was it just for tests ?

Reply:
I improves a little bit performance but yes in any case it has to be done in another PR. I revert.

Reply:
reverted